### ISSUE TITLE
Stable Snap Release Version Update

### ISSUE THREAD

**Issue by scottclk** [aws/amazon-ssm-agent#478]
Is there any documented policy on how frequent the stable snap version is updated? Stable snap has not been updated since [11th May 2022](https://snapcraft.io/amazon-ssm-agent). We would like to use the remote host port forwarding feature but prefer stable over candidate.

It also looks like the release notes are not correct as [latest stable snap release](https://github.com/aws/amazon-ssm-agent/releases/tag/3.1.1188.0) (3.1.1188.0) shows:
```
* Added support for port forwarding to remote host
```

Trying to use remote port forwarding on this version throws an error:
```sh
Error: An error occurred (BadRequest) when calling the StartSession operation: The version of the SSM agent installed on this instance doesn't support port forwarding to remote hosts. Install the latest version of the SSM agent to use port forwarding sessions to remote hosts.
```

Although [AWS Documentation](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-sessions-start.html#sessions-remote-port-forwarding) states that version 3.1.1374.0 is actually required so the release notes may just need updating:
> To start a Session Manager port forwarding session to a remote host, version 3.1.1374.0 or later of SSM Agent must be installed on the managed node. The remote host isn't required to be managed by Systems Manager.

It would also be nice to apply a candidate and stable tag for the releases to make it easier to identify.

**Comment by scottclk** [comment]
Closing Issue with AWS Support Response. 
**TLDR**; Stable Snap version was updated yesterday to 3.1.1732.0

>have taken a look at the GitHub post you created and have had a look at the link to [snapcraft.io](http://snapcraft.io/) and looking at amazon-ssm-agent I can see that it shows that it was last updated Yesterday and the latest/stable version is showing as 3.1.1732.0 which was released 24th August 2022 [1].
Regarding the release notes I have spoken with a Subject Matter Expert in SSM and he has explained that there are times when the service team will update the release notes with functions that have been added to the SSM agent however, they have not been fully released. Like for version 3.1.1188.0 as you said it shows it \"added support for port forwarding to remote host\" but when using this version of the agent and try to start a session an error is returned stating that this version does not support the feature.
This is because the actual support for the feature within SSM was not fully released until 27th May 2022 which can be seen in the following blog article regarding the features support [3]. I was able to test this in my own environment using the agent version 3.1.1188.0 and was unable to establish a port forwarding session I then upgraded the SSM agent version to 3.1.1374.0 and was able to successfully setup a forwarding session. This means that even though the feature was added in 3.1.1188.0 to the agent it is not enabled or supported to start a port forwarding session to a remote host, you will need to use 3.1.1374.0 or later [4].
Unfortunately there is no documentation regarding when or how frequently the amazon-ssm-agent stable snapshot version is updated however, I will open an internal ticket with the SSM service team to see if they can provide any additional information regarding this as well as asking if they can add the stable/candidate tag to the release notes. Please be aware that I cannot provide an ETA or any guarantee that the service team will add the additional tags to the release notes.
[1] - https://github.com/aws/amazon-ssm-agent/releases/tag/3.1.1732.0
[2] - https://github.com/aws/amazon-ssm-agent/releases/tag/3.1.1188.0
[3] - https://aws.amazon.com/about-aws/whats-new/2022/05/aws-systems-manager-support-port-forwarding-remote-hosts-using-session-manager/
[4] - https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-sessions-start.html#sessions-remote-port-forwarding
I hope you find the above information useful. If you have any questions, please feel free to reply through this case at any time.
